src/makeRobotAccountant.js

```diff
@@ -6,7 +6,17 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let count = 0;
+
+  return (firstNumber) => (secondNumber) => {
```

Comment:
Not the most readable syntax. Fix it.
```suggestion
  return (firstNumber) => {
    return (secondNumber) => {
```